src/Elastic.Markdown/IO/MarkdownFile.cs

```diff
@@ -46,7 +46,11 @@ public DocumentationGroup? Parent
 	public string? Title { get; private set; }
```

Comment:
Maybe introduce a `TitleRaw` property thats the markdown so that we can assume `Title` and `NavigationTitle` are sanitized. 

Then when rendering we can use `TitleRaw`

Reply:
I'm a bit unsure about this. This caused changes in various places, where it's not entirely clear to me if it expects an a raw markdown string or sanitized title. 

Was the assumption that it's only a sanitized title during the first implementations?

Reply:
See my comment here: https://github.com/elastic/docs-builder/pull/290/files#r1927019403

Reply:
https://github.com/elastic/docs-builder/commit/e00d555e7d3d962ca93f06798074c24b261c20fe